Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /app
 COPY ${JAR_LIB_FILE} app.jar
 
 # Run the api
-CMD ["java", "-jar", "app.jar"]
+CMD ["java", "-agentlib:jdwp=transport=dt_socket,address=*:5005,server=y,suspend=n", "-jar", "app.jar"]
 
```

Comment:
RS debug port is `5005` as well. This might create conflicts if running both applications, in docker, at the same time. Thoughts on changing to another port number?